mne/preprocessing/bads.py

```diff
@@ -5,12 +5,42 @@
 import numpy as np
 
 
-def _find_outliers(X, threshold=3.0, max_iter=2):
+def _find_outliers(X, threshold=3.0, max_iter=2, tail=0):
+    """Find outliers based on iterated Z-scoring.
+
+    This procedure compares the absolute z-score against the threshold.
+    After excluding local outliers, the comparison is repeated until no
+    local outlier is present any more.
+
+    Parameters
+    ----------
+    X : np.ndarray of float, shape (n_elemenets,)
```

Comment:
spelling error `n_elemenets`. But also not obvious what the "elements" are here since "elements" is not a function param, nor is it mentioned in the docstring.